Source/WebKit/UIProcess/API/Cocoa/WKMenuItemIdentifiers.mm

```diff
@@ -60,7 +60,7 @@
 NSString * const _WKMenuItemIdentifierSpeechMenu = @"WKMenuItemIdentifierSpeechMenu";
 
 NSString * const _WKMenuItemIdentifierTranslate = @"WKMenuItemIdentifierTranslate";
-NSString * const _WKMenuItemIdentifierWritingTools = @"WKMenuItemIdentifierWritingTools";
+NSString * const _WKMenuItemIdentifierWritingTools = @"__NSTextViewContextSubmenuIdentifierWritingTools"; // AppKit creates this menu item.
```

Comment:
Is there an AppKit bug about exposing this as a constant?

Reply:
nope, because there already is a constant; why I didn't think of using it, I do not know lol. will fix

Reply:
nvm, I'll file a bug